### ISSUE TITLE
sensitive is low for catch barcode

### ISSUE THREAD

**Issue by sam9010** [moaazsidat/react-native-qrcode-scanner#409]
when I show barcode on camera, sometimes it's sensitive is low for catch barcode( I must do my camera back and forth for catch) 
do you have any solution?

**Comment by midhungopipk** [comment]
Yeah same for QR code too. What must be the reason for this? are there any solutions?

QR scanning is a kind of core functionality in my app. That's why this issue worries me.

**Comment by sam9010** [comment]
no I could not find any solution, if you find any solution I am very glad to share this.

**Comment by Yukesh2412** [comment]
> Yeah same for QR code too. What must be the reason for this? are there any solutions?
> 
> QR scanning is a kind of core functionality in my app. That's why this issue worries me.

Hi, I suggest you to use a different package such as [react-native-vision-camera](https://github.com/mrousavy/react-native-vision-camera) or [vision-camera-code-scanner](https://github.com/mrousavy/vision-camera-code-scanner)
- Which use JSI and frame processor for QR code scanning. 
- These packages are faster and they don't require linking or permissions.

**Comment by sam9010** [comment]
@Yukesh2412  thanks a lot I test it as soon as I have time
